department-of-veterans-affairs/va.gov-cms
# VACMS-15062 add field_administration back to jsonapi responses

## Description
Relates to #15062. I was too eager removing things from the jsonapi response in the past. We will need `field_administration` available in order to properly check & generate pages under the Lovell umbrella.

## Testing done
- `/jsonapi/node/news_story`
- `/jsonapi/node/story_listing`
- `/jsonapi/node/person_profile`
- `/jsonapi/node/banner`
- `/jsonapi/node/promo_banner`
- `/jsonapi/node/full_width_banner_alert`

Should all properly include `field_administration` under the relationships key now.

## Screenshots
![Screenshot 2023-09-15 at 8 40 18 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/11279744/8c60713c-f965-4cd6-af50-6bce2112034e)

See the below link for the current behavior, which does not include `field_administration` in the data for a given node.
https://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/jsonapi/node/news_story

## QA steps
- Log into the generated tugboat PR environment
- visit the six jsonapi endpoints described above and verify that `field_administration` is included for each of them.

### Select Team for PR review
